src/index.js

```diff
@@ -542,7 +550,7 @@ export const Webform = ({
           onFinishFailed={onCompleteFailed}
           style={style}
         >
-          {formsMemo?.question_group.map((g, key) => {
+          {orderBy(formsMemo?.question_group, 'order')?.map((g, key) => {
```

Comment:
Hi @navins94 , 
Why are we not order by the question_group and question inside `formsMemo` ?

Reply:
Hi @wayangalihpratama,
Do you mean to say inside formsMemo callback ?